src/components/chat/MessageInput.js

```diff
@@ -0,0 +1,76 @@
+/* eslint react/jsx-no-bind: 0 */
+import React from 'react';
+import { createForm } from 'rc-form';
```

Comment:
👍

Reply:
Never used `rc-form`. Interesting, I find the usage not very react-redux friendly.